charts/workflow/values.yaml

```diff
@@ -130,6 +132,13 @@ logger:
     port: "redis port"
     password: "redis password" # "" == no password
 
+fluentd:
+  syslog:
+    # Configure the following ONLY if using Fluentd to send log messages to both
+    # the Logger component and external syslog endpoint
+    syslog_host: "external syslog endpoint url"
```

Comment:
can you keep the values as empty like `syslog_host: ""`  and add the help test as comments above so nothing gets applied by default as https://github.com/deis/fluentd/pull/72/files?diff=unified#diff-65e48c77b3de178d48580127e51490c5R36 becomes true if the string is not empty.